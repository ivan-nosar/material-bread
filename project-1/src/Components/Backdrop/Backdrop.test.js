import React from 'react';
import { Backdrop } from '../../';

import ShallowRenderer from 'react-test-renderer/shallow';
const renderer = new ShallowRenderer();

test('Backdrop Renders', () => {
  const tree = renderer.render(<Backdrop />);
  expect(tree).toMatchSnapshot();
});
