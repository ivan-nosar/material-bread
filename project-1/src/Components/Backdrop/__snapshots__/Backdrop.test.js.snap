// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop Renders 1`] = `
<Context.Consumer>
  [Function]
</Context.Consumer>
`;
